src/collective/jekyll/browser/collection_view.pt

```diff
@@ -44,38 +43,45 @@
                                      tal:replace="structure symptom/@@status" />
                             </td>
                             </tal:symptoms>
-                            <tal:fields repeat="field here/getCustomViewFields"
+                            <tal:fields repeat="field here/customViewFields"
                                         define="normalizeString nocall:context/@@plone/normalizeString;
+                                                toLocalizedTime nocall: context/@@plone/toLocalizedTime;
                                                 url obj/getURL;
                                                 obj_type obj/portal_type;
+                                                obj_type_class     python:'contenttype-' + normalizeString(obj_type);
                                                 obj_description obj/Description;
                                                 obj_typeinfo python: here.portal_types.getTypeInfo(obj_type);
-                                                obj_icon python:plone_view.getIcon(obj);
+                                                obj_has_image      python:obj.getIcon;
                                                 obj_wf_state obj/review_state|python: wtool.getInfoFor(obj, 'review_state', '');
                                                 obj_wf_state_class python:'state-' + normalizeString(obj_wf_state);
                                                 obj_url python:url+'/view' if obj_type in use_view_action else url;
                                                 title_or_id obj/pretty_title_or_id">
                             <tal:block define="value python: getattr(obj, field, None)">
-                                <td tal:condition="python: field != 'Title'"
-                                    tal:content="python:here.formatCatalogMetadata(value)"
+                                <td tal:condition="python: field != 'Title' and field not in ['ModificationDate',
+                                        'CreationDate', 'EffectiveDate',
+                                        'ExpirationDate', ]"
+                                    tal:content="value"
```

Comment:
I prefer logic to be kept in view classes. Would it be possible to move this `tal:condition` test to a method on a view class ?